src/ship/PlayerShipController.cpp

```diff
@@ -498,19 +495,22 @@ void PlayerShipController::StaticUpdate(const float timeStep)
 	if (m_ship->GetFlightState() == Ship::FLYING) {
 		switch (m_flightControlState) {
 		case CONTROL_FIXSPEED:
-			PollControls(timeStep, true, mouseMotion, act);
+			PollControls(timeStep, mouseMotion, act);
 			FlightAssist(timeStep, act);
 			ApplyTotalAction(act);
 			break;
 		case CONTROL_MANUAL:
 			// maintain the thrust set by the station when undocking while the timeout lasts
 			// but only in manual mode
-			PollControls(timeStep, true, mouseMotion, act);
+			PollControls(timeStep, mouseMotion, act);
 			Util::LimitActualSpeed(*this, timeStep, act);
 			// synchronize the rotation in 'follow orient' mode
 			if (m_followTarget && m_followMode == FOLLOW_ORI) {
 				act.desiredAngular += m_followTarget->GetAngVelocity() * Pi::player->GetOrient();
 			}
+			//Follow mode as I understand is when we try to match rotation speeds with a target
+			//so all axes should be matched even with rotation damping off 
```

Comment:
I would like to add that the ship not only corrects its orientation, but also corrects its position in order to remain "stationary" relative to the turning target.
Therefore, I personally believe that in this case the rotation can be "released", but it is also possible not to do so.

Reply:
If you fly out of a space station and turn on the "fallow orientation" mode you will match the rotation with the space station but the ship will be still drifting out into space and you will be in this FALLOW_ORI mode internally. With rotation dampening off I think we want the same behavior.
There is another mode FALLOW_POS that is maybe what you describe.

Reply:
Ok, if you do not mind then I will leave it as it is now. Rotation damping off is a minor thing in my opinion.